learn-beta2.md

```diff
@@ -10,7 +10,7 @@ redirect_from:
 ---
 ## Getting Started
 
-### Language Introduction Video
+### [Language Introduction Video](/learn/language-concepts/Ballerina_Swan_Lake_Presentation_Deck_V1.0.pdf)
```

Comment:
Why are we adding a PDF for a Video topic?

Reply:
Updated.